.azure/pipelines/ci.yml

```diff
@@ -577,9 +577,20 @@ stages:
       agentOs: Windows
       timeoutInMinutes: 180
       steps:
-      - script: .\restore.cmd -ci
-        displayName: Restore
-      - script: .\build.cmd -ci -NoRestore -test -projects eng\helix\helix.proj /p:IsRequiredCheck=true /p:IsHelixJob=true /p:BuildAllProjects=true /p:BuildNative=true /p:ASPNETCORE_TEST_LOG_DIR=artifacts/log -bl
+      # Build the x86 shared framework
+      # TODO: make it possible to build for one Windows architecture at a time
```

Comment:
Don't check in TODO comments, file bugs

Reply:
I'm just copying this as is from: https://github.com/dotnet/aspnetcore/blob/master/.azure/pipelines/ci.yml#L123

Reply:
Please fix that as part of this PR i.e. remove the comment. Not sure whether the comment is applicable anymore anyhow. If it is, there's probably already an 'area-infrastructure' issue about it in this repo or dotnet/aspnetcore-internal -- should be at least.